### ISSUE TITLE
The start new scan button is not accessible by screen reader

### ISSUE THREAD

**Issue by yongshing32** [greenbone/gsa#4788]
To start a new scan, according to your documentation, on web interface, we go to scan > task and move mouse over the 'new' and new task. As a screen reader user, I simply can't find the 'new task' as mentioned as screen reader relies on keyboard navigation. 

### Expected behavior

Would be able to find the 'new' button and activate the 'new task' button/link using screen reader.

### Actual behavior

When I go to the page scan > task, I don't find the 'new' button or the 'new task' or 'new wizard'. 

### Steps to reproduce

1. Login to web interface 
2. Go to scan > task 
3. Supposedly we can find the 'new' or 'new task' or 'new wizard' but for screen reader user, we unable to reach those buttons-- so they are not accessible

### GVM versions

**gsa:** Greenbone Security Assistant 24.2.3~git                                                                                 

**gvm:** Greenbone Vulnerability Manager 26.0.0                                                                                 

### Environment

Gvm installed in kali linux 

### Logfiles

not relevant to the issue of accessibility with screen reader
